test/selenium/configurations.rb

```diff
@@ -127,6 +127,12 @@ def configure_for_chrome_web(args)
   common_configuration_for_web(args)
 end
 
+def configure_for_safari_web(args)
+  common_configuration_for_web(args)
+  # Do not block pop-up windows
+  `defaults write com.apple.Safari com.apple.Safari.ContentPageGroupIdentifier.WebKit2JavaScriptCanOpenWindowsAutomatically -bool true`
```

Comment:
https://github.com/sammyshj/privly-applications/blob/updateselenium/test/selenium/configurations.rb#L133 enables pop-up windows in Safari.
This can be done by opening Safari and navigating to, Preferences -> Security -> Uncheck block pop-up windows.
But I thought that rather than the user doing the whole thing, we can simply run this from the script itself.
Do you think this line should be removed or should be put into some other file?